webpack/scenes/RedHatRepositories/index.scss

```diff
@@ -73,3 +73,15 @@
     }
   }
 }
+
+.sticky-pagination {
+  position: sticky;
```

Comment:
According to [caniuse](https://caniuse.com/#feat=css-sticky) this may not be supported in all of our supported browsers.  Any idea what the fallback would be?  I'd assume that it would just stay at the top of the table when you scroll.

Reply:
AFAIK it would stay static on unsupported browsers (IE11...), therefore the pagination won’t be broken .